src/Theme/Global.elm

```diff
@@ -178,6 +180,16 @@ globalStyles =
 -- Helpers
 
 
+primaryHeader : String -> Html Msg
+primaryHeader title =
+    h1 [ css [ primaryHeaderStyle ], id "focus-target" ] [ text title ]
```

Comment:
This is a good idea! It looks like you're just missing the `tabindex -1` attribute Katja had in her example

Reply:
added `tabindex`